src/main/java/org/broadinstitute/hellbender/utils/genotyper/ReadLikelihoods.java

```diff
@@ -1078,6 +1078,39 @@ private void removeSampleReads(final int sampleIndex, final IntArrayList indexTo
     }
 
     /**
+     * Transform into a multi-sample HashMap backed {@link PerReadAlleleLikelihoodMap} type.
+     * @return never {@code null}.
+     *
+     */
+    public Map<String, PerReadAlleleLikelihoodMap> toPerReadAlleleLikelihoodMap() {
```

Comment:
Let's not do it right now. Too much pain - that class is everywhere. I entered https://github.com/broadinstitute/gatk/issues/1467